third_party/terraform/tests/resource_logging_project_sink_test.go

```diff
@@ -241,6 +241,43 @@ func TestLoggingProjectSink_bigqueryOptionCustomizedDiff(t *testing.T) {
 	}
 }
 
+func TestAccLoggingProjectSink_disabled_update(t *testing.T) {
+	t.Parallel()
+
+	sinkName := "tf-test-sink-" + randString(t, 10)
+	bucketName := "tf-test-sink-bucket-" + randString(t, 10)
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckLoggingProjectSinkDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccLoggingProjectSink_disabled_update(sinkName, getTestProjectFromEnv(), bucketName, "true"),
```

Comment:
The issue specifies that it can't go from false -> true, but this test is for true -> false. Should we add a third step that sets it to true again?

Reply:
The root cause is the field can't be updated which was reflected in the PR subject (the issue subject is just updated accordingly)  but I have added a third step anyway. 
Reformatting the changelog is done